src/Maybe.js

```diff
@@ -95,4 +97,12 @@ _Nothing.prototype.getOrElse = function(a) {
   return a;
 };
 
+_Just.prototype.toString = function() {
+  return 'Maybe.Just(' + R.toString(this.value) + ')';
+};
+
+_Nothing.prototype.toString = function() {
+  return 'Maybe.Nothing()';
```

Comment:
The idea is that `eval(R.toString(x))` should be equivalent to `x` for every `x`.

It's up to you, though. I'll remove the parens if you'd prefer the string representation to omit them.